indent/ps1.vim

```diff
@@ -1,20 +1,87 @@
 " Vim indent file
-" Language:           Windows PowerShell
-" Maintainer:         Peter Provost <peter@provost.org>
-" Version:            2.10
-" Project Repository: https://github.com/PProvost/vim-ps1
-" Vim Script Page:    http://www.vim.org/scripts/script.php?script_id=1327"
+" Language:       Windows PowerShell
+" Maintainer:
+"   Lior Elia - http://blogs.msdn.com/lior
+"   Crane Jin <crane@cranejin.com>
+" Version:        2.0.1
+" Last Change:    2016 Aug 10
+" Changelist:
+"    1.0     2009-10-04    LiorE       Comment rows
+"    2.0     2009-10-18    LiorE       Lines after (.*), comments after '{' or
+"                                      '}' and many many other things. I've made
+"                                      this a major version change because there
+"                                      are many changes in the behavior.
+"    2.0.1   2016-08-10    cranej      Remove debug code
+
+" Dedicated to my loving and supporting family: Anat, Amit and Ofir.
```

Comment:
This script is not written by me.  I just want to integrate it into this repo so we don't need to use two separate plugins for PowerShell.  So to respect the copyright,  I think it may be better to keep 'loving and supporting' and other headers which were added by author.